### DISCUSSION TITLE
Errata corrige

### DISCUSSION THREAD

**Discussion by ca1ig0la** [Ditectrev/AWS-Certified-Solutions-Architect-Associate-SAA-C03-Practice-Tests-Exams-Questions-Answers#23] [General]
Hi all,
I am finding this resource very useful. Thanks.
I think answer to question **"34. Multi-AZ deployment [...] supported for Microsoft SQL Server DB Instances"** might be outdated: https://docs.aws.amazon.com/it_it/AmazonRDS/latest/UserGuide/USER_SQLServerMultiAZ.html

**Comment by danieldanielecki** [comment]
@ca1ig0la, kindly please submit an issue, or a pull request with the correct answer ;-)
